tests/test_java.py

```diff
@@ -219,7 +219,7 @@ def test_java_safemode_build_number_check(
     "in_container,java_args,check_app_exited",
     [
         (False, [], False),  # default
-        (False, ["-XX:ErrorFile=/tmp/my_custom_error_file.log"], False),  # custom error file
+        (False, ("-XX:ErrorFile=/tmp/my_custom_error_file.log",), False),  # custom error file
         (True, [], False),  # containerized (other params are ignored)
```

Comment:
Please modify the java_args to be a tuple everywhere (it fails the tests)

Reply:
Missed it. Anyway I'm not sure if that's the only cause, let's see what's failing next